gcloud_utils/bigquery/bigquery.py

```diff
@@ -1,4 +1,4 @@
-#pylint: disable=too-many-arguments,bare-except
+#pylint: disable=too-many-arguments,bare-except, broad-except
```

Comment:
why do you remove this lint?

Reply:
```
def create_table(self, dataset_id, table_id):
    """Create a table based on dataset"""
    try:
        self.create_dataset(dataset_id)
    except Exception:
        pass
```

@dmvieira It's a temporary removal until I know what the real type of exception is going to be caught here. Pylint accuses the use of `except: Exception`. But if you know what type I can fix.

Reply:
@alexandreyy do you know about this Exception?

Reply:
@dmvieira No. We should remove that code or handle that Exception.

Reply:
So, what should I do?

Reply:
I prefer to remove it. @dmvieira What do you think?

Reply:
@alexandreyy remove the create_table function?

Reply:
@hericlesme The Exception

Reply:
@alexandreyy Okay. Can I do this? Or should I wait for @dmvieira?

Reply:
@dmvieira @hericlesme wait :)